samtranslator/model/sam_resources.py

```diff
@@ -1778,6 +1778,7 @@ class SamStateMachine(SamResourceMacro):
         "PermissionsBoundary": PropertyType(False, IS_STR),
         "AutoPublishAlias": PassThroughProperty(False),
         "DeploymentPreference": MutatedPassThroughProperty(False),
+        "UseAliasAsEventTarget": Property(False, IS_BOOL),
```

Comment:
Should we support this property in globals section?

Reply:
Could be a nice improvement, let's see if customers will be asking for it. For now I don't think this should be blocking this fix.